docs/multicluster/quick-start.md

```diff
@@ -3,6 +3,10 @@
 In this quick start guide, we will set up an Antrea Multi-cluster ClusterSet
 with two clusters. One cluster will serve as the leader of the ClusterSet, and
 meanwhile also join as a member cluster; another cluster will be a member only.
+Antrea Multi-cluster supports different types of endpoints as the multi-cluster
```

Comment:
different types -> two types

Reply:
types of endpoints -> types of IP addresses?

Reply:
the multi-cluster Service endpoints -> multi-cluster Service endpoints

Reply:
Done